### ISSUE TITLE
[Suggestion] Deleting connections between nodes

### ISSUE THREAD

**Issue by keli5** [ZakBlystone/gmod_blueprints#103]
Maybe I'm missing something, but a way to delete connections between nodes without deleting and recreating one node would be nice to have. As it stands, seemingly the only way to disconnect two nodes is to delete one of them?

**Comment by ZakBlystone** [comment]
Hi, yes there is a way to disconnect nodes without deleting them...
Hold Ctrl while clicking on a connected pin, then drag the connection off to disconnect it.

Hope this helps :)

**Comment by keli5** [comment]
Thanks :)
